specification/policyinsights/resource-manager/Microsoft.PolicyInsights/preview/2018-07-01-preview/policyStates.json

```diff
@@ -1165,11 +1165,11 @@
         },
         "expressionValue": {
           "description": "Value of the expression.",
```

Comment:
Have you already generated any client SDKs for the existing version? This looks like a breaking change. Will your service continue to accept strings?

Reply:
SDKs for other languages are published but not SDK for .NET. This is part of the response not the request.

Reply:
I see, so this change in behavior from your RP may break users of these existing SDKs, as response deserialization may fail. Normally a breaking change would require an API version rev. I'll discuss with the team. Is this feature in public preview or GA?

Reply:
There is no change in the RP behavior. The RP has been always returning "any" type for these properties, which is expected. The issue is in the client code generation since the current swagger spec has incorrect type for these properties and the user of the latest SDK may see a failure when the property has a non-string value. So the PR should be treated as a fix not a breaking change.

The PR is in GA. The feature is only available in the preview API version.